com.sejda.Desktop.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.sejda.Desktop</id>
+  <name>Sejda PDF Desktop</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Sejda BV</developer_name>
+  <summary>Productive PDF software that you'll love to use</summary>
```

Comment:
This is marketing speak. Not descriptive.

Reply:
I changed it